ansible/files/bootstrap/python-requirements.txt

```diff
@@ -12,6 +12,7 @@ py4j==0.8.2.1
 pyliblzma==0.5.3
 python-mozaggregator==0.3.0.1
 python_moztelemetry>=0.5,<1.0.0
+pyspark-hyperloglog==2.1.1
```

Comment:
The python bindings are packaged in the jar we pull from spark-packages.org, but unfortunately EMR has strange behavior such that pyspark sessions aren't able to see the python code (see https://bugzilla.mozilla.org/show_bug.cgi?id=1466936#c7)

By installing the python package here, folks will be able to `import pyspark_hyperloglog` at spinup without having to hack `sys.path`.

I have an issue open with AWS to discuss whether there's a long-term solution for accessing python code held in spark packages.